File: Problems/week-1/1/twosum.go
func twoSum(nums []int, target int) []int {
    var result []int
    for i, value := range nums {
        for j := i + 1; j < len(nums); j++ {
            if int64(value) + int64(nums[j]) == int64(target) {
                result = []int{i, j}
                return result
            }
        }
    }
    return result
}

File: Problems/week-1/1/twosum.ts
function twoSum(nums: number[], target: number): number[] {
    var result: number[];
    for (var i = 0, len = nums.length, find = false; i < len && !find; i++) {
        for (var j = i + 1; j < len && !find; j++) {
            if (nums[i] + nums[j] == target) {
                result = [i, j];
                find = true;
            };
        };
    };
    return result
};
